portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -14,19 +14,57 @@
 
 package com.google.sps.servlets;
 
+import com.google.sps.data.QuotePerson;
+import com.google.gson.Gson;
 import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+/** Servlet that returns a random quote from the office. */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  private List<QuotePerson> quotes;
+
+  /**
+   * Add all the quotes to quotes using type QuotePerosn on start.
+   */
+  @Override
+  public void init() {
+    quotes = new ArrayList<>();
+    quotes.add(new QuotePerson("Bears. Beets. Battlestar Galactica.", "Jim Halpert"));
+    quotes.add(new QuotePerson("I'm not supersitious, but I am a little stitious.", "Michael Scott"));
```

Comment:
LOL just saw this episode a few days ago.

Reply:
It's a great quote!